buildpack/dockerfile.go

```diff
@@ -17,7 +17,7 @@ const (
 	DockerfileKindRun   = "run"
 )
 
-var permittedCommandsBuild = []string{"from", "add", "arg", "copy", "env", "label", "run", "shell", "user", "workdir"}
+var permittedCommandsBuild = []string{"FROM", "ADD", "ARG", "COPY", "ENV", "LABEL", "RUN", "SHELL", "USER", "WORKDIR"}
```

Comment:
Docker uses a newer version of buildkit - I tried upgrading, but it breaks kaniko. However this change (which was needed when buildkit was upgraded) seems nice to have anyway.